pkg/sql/colexec/agg/stddevpop.go

```diff
@@ -104,7 +105,7 @@ func (s *StdD64) Grows(size int) {
 func (s *StdD64) Eval(vs []types.Decimal128, err error) ([]types.Decimal128, error) {
 	vs, err = s.Variance.Eval(vs, err)
 	if err != nil {
-		return nil, err
+		return nil, moerr.NewOutOfRangeNoCtx("Decimal64", "Stddevpop")
```

Comment:
Why not return nil, err?

Whatever eval generate, should be good or better for upper layer.